src/components/radio/RadioGroup.js

```diff
@@ -47,20 +47,24 @@ class RadioGroup extends React.Component<Props, State> {
 
     render() {
         const { children, className, name } = this.props;
-        const { value } = this.state;
 
         return (
             <div className={`radio-group ${className}`} onChange={this.onChangeHandler}>
-                {React.Children.map(children, radio => (
-                    <RadioButton
-                        description={radio.props.description}
-                        isDisabled={radio.props.isDisabled}
-                        isSelected={radio.props.value === value}
-                        label={radio.props.label}
-                        name={name}
-                        value={radio.props.value}
-                    />
-                ))}
+                {React.Children.map(children, radio => {
+                    const { description, isDisabled, label, value, ...rest } = radio.props;
+
+                    return (
+                        <RadioButton
+                            description={description}
```

Comment:
If you are using ...rest, you dont need to explicitly add anything but name and isSelected